scripts/linux/runE2ETest.sh

```diff
@@ -714,7 +718,7 @@ function run_longhaul_test() {
         --initialize-with-agent-artifact "$INITIALIZE_WITH_AGENT_ARTIFACT" \
         --leave-running=All \
         -l "$deployment_working_file" \
-        --runtime-log-level "Info" \
+        --runtime-log-level "$RUNTIME_LOG_LEVEL" \
```

Comment:
--runtime-log-level parameter is used in stress test as well.  Please update to use $RUNTIME_LOG_LEVEL.

Reply:
sure